### PR TITLE
Add instructions for Publish Catalog Template/Deploy Catalog App steps

### PR DESCRIPTION
**Pull request by gitlawr**
Address issue: https://github.com/rancher/docs/issues/1017

### CODE REVIEW

**Comment by loganhz** [comment]
LGTM

**Comment by loganhz** [comment]
Please add `_Available as of v2.2.0_` so that people know when these features were available.

https://github.com/rancher/docs/pull/1199/files#diff-8ad9c71b76b83c64e18dda58fa1e8c68R9

File: content/rancher/v2.x/en/tools/pipelines/configurations/_index.md [hunk 1]
```diff
@@ -187,6 +187,60 @@ PLUGIN_BUILD_ARGS       | Docker build args, a comma separated list
...
+{{% /tab %}}
+
+{{% tab "By YAML" %}}
+
+```yaml
+# example
+stages:
+- name: Publish Wordpress Template
+  steps:
+  - publishCatalogConfig:
```
Comment (loganhz) [inline content/rancher/v2.x/en/tools/pipelines/configurations/_index.md]: Can you add some comments in the yaml like what we do in https://github.com/rancher/docs/pull/1037/files

Comment (gitlawr) [reply content/rancher/v2.x/en/tools/pipelines/configurations/_index.md]: Updated

**Comment by loganhz** [comment]
LGTM

File: content/rancher/v2.x/en/tools/pipelines/configurations/_index.md [hunk 2]
```diff
@@ -187,6 +187,74 @@ PLUGIN_BUILD_ARGS       | Docker build args, a comma separated list
...
+
+
+* Path: The relative path to the chart folder in the source code repo, where the `Chart.yaml` file is located.
+* CatalogTemplate: The name of the template.
+* Version: The version of the template you want to publish, it should be consistent with the version defined in the `Chart.yaml` file.
+* GitUrl: The git URL of the chart repository that the template will be published to.
+* GitBranch: The git branch of the chart repository that the template will be published to.
+* GitAuthor: The author name used in the commit message.
+* GitEmail: The author email used in the commit message.
+* Credentials: You should provide Git credentials by referencing secrets in pipeline dedicated namespace. If you publish via SSH protocol, inject your deploy key to the `DEPLOY_KEY` environment variable. If you publish via HTTP(S) protolcol, inject your username and password to `USERNAME` and `PASSWORD` environment variables.
```
Comment (ankur111) [inline content/rancher/v2.x/en/tools/pipelines/configurations/_index.md]: change " in pipeline dedicated namespace" to  "in dedicated pipeline namespace"

Comment (gitlawr) [reply content/rancher/v2.x/en/tools/pipelines/configurations/_index.md]: Updated

**Pull request merged**